sql/npc_list.sql

```diff
@@ -12849,7 +12849,7 @@ INSERT INTO `npc_list` VALUES (17207824,'qm4','???',102,426.315,8.563,-163.337,1
 INSERT INTO `npc_list` VALUES (17207825,'Monberaux','Monberaux',0,0.000,0.000,0.000,0,40,40,0,1,0,6,27,0x0000260300000000000000000000000000000000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17207826,'Wolfgang','Wolfgang',0,0.000,0.000,0.000,0,40,40,0,1,0,6,27,0x0100020114101920193019401950006000700000,32,NULL,1);
 -- airship
--- INSERT INTO `npc_list` VALUES (17207827,'','',0,0.000,0.000,0.000,0,40,40,19,0,0,2,3,0x040000000B000000000000000000000000000000,32,null,1);
+INSERT INTO `npc_list` VALUES (17207827,x'0B','NPC',0,0.000,200.000,0.000,65537,50,50,19,0,0,0,3,0x0400000000000000000000000000000000000000,0,NULL,1);
```

Comment:
Stray x outside of single quotes.

Reply:
That's intentional, x'0B' = character value of 0x0B, it's used for non-printable NPC names, see https://github.com/LandSandBoat/server/pull/2483 for a more obvious example of what this means

Reply:
Thanks for the clarification, first time I've come across it so it stood out as a potential typo to me.

Reply:
Oh yeah, it certainly is weird isn't it? It took me a good while to figure out how to use non-printable characters in the db. Very counter-intuitive compared to most other languages that lets you escape in string such as "\x0B"